### PR TITLE
Feature/e UI 3563 reassign page phillip whitaker

### PR DESCRIPTION
**Pull request by phillip-whitaker**
image

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3563

### Change description ###

Updated the reassign component to match requirements for release-2

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts [hunk 1]
```diff
@@ -51,26 +47,8 @@ export class TaskAssignmentContainerComponent implements OnInit {
...
-  };
   public ngOnInit(): void {
-    // Set up the default sorting.
-    this.sortedBy = {
-      fieldName: this.taskServiceConfig.defaultSortFieldName,
-      order: this.taskServiceConfig.defaultSortDirection
-    };
-
     // Get the task from the route, which will have been put there by the resolver.
     const task = this.route.snapshot.data.taskAndCaseworkers.task.task;
```
Comment (kuda1992) [inline src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts:53]: Can we have null checking?

File: src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts [hunk 2]
```diff
@@ -51,26 +47,8 @@ export class TaskAssignmentContainerComponent implements OnInit {
     return url || '/tasks/list';
```
Comment (kuda1992) [inline src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts:47]: link needs to be updated to `/mywork/list`

**Review by kuda1992 — CHANGES_REQUESTED** [review]

File: src/work-allocation-2/containers/task-assignment/task-assignment-container.component.html [hunk 3]
```diff
@@ -1,24 +1,15 @@
...
-
-      <h2 class="govuk-heading-m">Choose a colleague</h2>
-      <exui-task-assignment [excludeCaseworkers]="excludedCaseworkers" [location]="location"
-        (caseworkerChanged)="onCaseworkerChanged($event)" [showProblem]="error">
-      </exui-task-assignment>
-
-      <button class="govuk-button govuk-!-margin-right-1" type="submit" (click)="assign()">{{verb}}</button>
-      <button class="govuk-button govuk-button--secondary" type="submit" (click)="cancel()">Cancel and go back</button>
+      <h2 class="govuk-heading-m">Find the person</h2>
+      <p>Type the name of the person and select them</p>
```
Comment (paul-graham) [inline src/work-allocation-2/containers/task-assignment/task-assignment-container.component.html:9]: Needs the actual component for typing and selecting in here, surely...? 🤷‍♂️

**Review by paul-graham — CHANGES_REQUESTED** [review]
Looks to me like this cannot be completed until https://tools.hmcts.net/jira/browse/EUI-3568 is completed and `<find-the-person (onchange)="personFound($event)"></find-the-person>` (or whatever) can be placed inside `task-assignment-container.component.html`. 🤷‍♂️

**Comment by phillip-whitaker** [comment]
Closing because the branch I was told to branch off has change and the destination is different which has caused loads of conflicts....

**Pull request closed without merging**